requirements.txt

```diff
@@ -1,3 +1,2 @@
-pcaspy>=0.7.2
-PRUserial485>=2.5.0
+pcaspy>=0.7.3
 siriuspy>=2.2.0
```

Comment:
maybe it is interesting to define mathphys minimum version here

Reply:
done